Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL2luZm9ybWF0aWNhLmRhdGFtYW5hZ2VtZW50L29yZ2FuaXphdGlvbnM=/2024-05-08.xml

```diff
@@ -7,8 +7,12 @@
       <resource id="/subscriptions/{}/providers/informatica.datamanagement/organizations" version="2024-05-08" swagger="mgmt-plane/informatica/ResourceProviders/Informatica.DataManagement/Paths/L3N1YnNjcmlwdGlvbnMve3N1YnNjcmlwdGlvbklkfS9wcm92aWRlcnMvSW5mb3JtYXRpY2EuRGF0YU1hbmFnZW1lbnQvb3JnYW5pemF0aW9ucw==/V/MjAyNC0wNS0wOA=="/>
       <resource id="/subscriptions/{}/resourcegroups/{}/providers/informatica.datamanagement/organizations" version="2024-05-08" swagger="mgmt-plane/informatica/ResourceProviders/Informatica.DataManagement/Paths/L3N1YnNjcmlwdGlvbnMve3N1YnNjcmlwdGlvbklkfS9yZXNvdXJjZUdyb3Vwcy97cmVzb3VyY2VHcm91cE5hbWV9L3Byb3ZpZGVycy9JbmZvcm1hdGljYS5EYXRhTWFuYWdlbWVudC9vcmdhbml6YXRpb25z/V/MjAyNC0wNS0wOA=="/>
       <argGroup name="">
-        <arg type="ResourceGroupName" var="$Path.resourceGroupName" options="resource-group g" idPart="resource_group"/>
-        <arg type="SubscriptionId" var="$Path.subscriptionId" options="subscription" required="True" idPart="subscription"/>
+        <arg type="ResourceGroupName" var="$Path.resourceGroupName" options="rg" group="" idPart="resource_group">
```

Comment:
Please don't change the options of `resource-group g`. The rest commands use that options as a convention.

Reply:
I have reverted the change